PlainTasks.py

```diff
@@ -525,8 +542,11 @@ def show_panel_or_open(self, fn, sym, line, col, text):
                     name = os.path.join(root, fn)
                     if os.path.isfile(name):
                         self._current_res.append((name, line or 0, col or 0))
-            if os.path.isfile(fn): # check for full path
+            if os.path.isfile(fn):  # check for full path
                 self._current_res.append((fn, line or 0, col or 0))
+            elif os.path.isdir(fn):
+                self._open_folder(fn)
+                return
```

Comment:
I’m not sure about this.
It seems the right way to do it is append all matching directories (since we support relative paths, hence _all_) to `self._current_res` and then decide what must be done in `self._on_panel_selection`, i.e. open folder or file.

Also `self._format_res` should make it clear which item is file and which is folder and that folder would be added to project.
Also this feature must be available/visible only for ST3 users, since ST2 has no corresponding API.

Would be nice if you squashed excessive commits.